aaronurkin/microblog
# Add basic UI styles

## Summary
We want to deliver to our clients a user friendly and responsive UI. To achieve this, the application must be styled with a modern responsive design

## Test
1. Start the application server
2. Go to the home page, *eg., http://localhost:3000/*
3. Try to change your browser size to check the responsiveness
